templates/search/blocks/search_filters.html

```diff
@@ -106,7 +112,7 @@ <h4 class="tna-heading search-filters__accordion-section-heading" id="record_cov
 
                                 </div>
 
-                                <input type="submit" value="Update" class="search-filters__submit">
+<!--                                <input type="submit" value="Update" class="search-filters__submit">-->
```

Comment:
This comment is  starting from 1st col, it could be aligned under closing div to facilitate collapsing of html attributes in the editor.